server.ts

```diff
@@ -27,10 +29,34 @@ const client = new Client(dbConfig);
 client.connect();
 
 app.get("/", async (req, res) => {
-  const dbres = await client.query('select * from categories');
+  const dbres = await client.query('select * from pastedata limit 10');
   res.json(dbres.rows);
+  
 });
 
+app.post("/", async (req,res)=> {
+  try {
+    const input = req.body
+    const dbres = await client.query('insert into pastedata(title,data,creation_date) values($1,$2,$3) returning *',[input.title,input.data,input.creationDate])
```

Comment:
Reads nicer to destructure the body on line 39 and call variable names in the parameter array here